src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import "./Introduce.scss";
+import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+
+const Introduce = () => {
+
+    const navigate = useNavigate();
+
+    const goToProduct = () => {
+        navigate('/product');
+    };
+
+    return (
+    <div className = "box">
```

Comment:
```suggestion
    <div className = "introduce">
```
scss를 사용할 때는 컴포넌트의 가장 최상위 부모 태그는 컴포넌트의 이름과 동일하게 구현해주세요!